notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>I would remove the first point, it's too generic and makes me think "why should I bother then?". Let's focus on the second point, that's very important instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>done</p>